gateway/api/guardrails/guardrails.go

```diff
@@ -86,23 +97,32 @@ func Put(c *gin.Context) {
 		Output:      req.Output,
 		UpdatedAt:   time.Now().UTC(),
 	}
-	err := models.UpdateGuardRailRules(rule)
+
+	log.Infof("Updating guardrail %s with connections: %v", rule.ID, req.ConnectionIDs)
```

Comment:
Remove this log statement please, it's not necessary since we're already auditing this endpoint in a middleware